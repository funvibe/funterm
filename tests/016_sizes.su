lua.sizes = <<1:4, 15:4, 255:8, 1000:16, 100000:32>>

match lua.sizes {
    <<a:4, b:4, c:8, d:16, e:32>> -> {
        lua.print("4-bit values:", a, b)
        lua.print("8-bit value:", c)
        lua.print("16-bit value:", d)
        lua.print("32-bit value:", e)
    },
    _ -> lua.print("Match failed")
}