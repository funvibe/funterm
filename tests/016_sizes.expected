4-bit values: 1 15
8-bit value: 255
16-bit value: 1000
32-bit value: 100000